docs/astro/src/content/docs/guide/tooling/figma-inspector.mdx

```diff
@@ -0,0 +1,88 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Figma Variable Export
+description: Exporting Figma Variables to Slint Design Tokens
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import LangRefLink from '/src/components/LangRefLink.astro';
+
+## Setting up the Figma Inspector
```

Comment:
Perhaps before setting up, there could be an intro section that briefly explains what the problem space is and what our solution roughly entails?

*After* that, a section that explains how to set up the inspector makes sense.